### PR TITLE
Ability to yell multiple times

### PR DESCRIPTION
**Pull request by johnsgill3**
I limited it to only yell 10 times, and then reply to the user telling them that's not nice...

### CODE REVIEW

File: scripts/games.js [hunk 1]
```diff
@@ -1130,7 +1131,17 @@ module.exports = function(robot) {
         })){
             return msg.reply(matchFormat('It\'s not their turn',msg));
         }
-        return robot.messageRoom(gameRoom, matchFormat(formatMessage(user).message,msg));
+        if(count === undefined) {
+            return robot.messageRoom(gameRoom, matchFormat(formatMessage(user).message,msg));
+        } else {
```
Comment (gcochard) [inline scripts/games.js]: No need for else, since the if path returns.

File: scripts/games.js [hunk 2]
```diff
@@ -1130,7 +1131,17 @@ module.exports = function(robot) {
...
+            var c = 1;
+            for(; c <= 10 && c <= count; c++) {
+                robot.messageRoom(gameRoom, matchFormat(formatMessage(user).message,msg));
+            }
+            if (c != count) {
```
Comment (gcochard) [inline scripts/games.js]: Move this up above the loop and return, we should short-circuit bad behavior.

Comment (johnsgill3) [inline scripts/games.js]: So you don't want the yell if n > 10 or whatever limit we want?

Comment (gcochard) [inline scripts/games.js]: Yes, our hubot is extremely opinionated and pedantic.

**Pull request merged**